scripts/install-worker.sh

```diff
@@ -121,11 +122,7 @@ if [[ "$INSTALL_DOCKER" == "true" ]]; then
     sudo mv $TEMPLATE_DIR/docker-daemon.json /etc/docker/daemon.json
     sudo chown root:root /etc/docker/daemon.json
 
-    # https://github.com/awslabs/amazon-eks-ami/issues/563
-    # Due to an issue with the latest containerd, customers are seeing
-    # pods getting stuck in terminating, so we need to downgrade containerd
-    # until the issue is fixed upstream or we have another workaround.
-    sudo yum downgrade -y containerd-1.3.2-1.amzn2
+    sudo yum downgrade -y containerd-${CONTAINERD_VERSION}
```

Comment:
Will this work if you pass in either a higher or the same version of containerd?

Reply:
This won't work if we pass in a higher version than what's already installed.

This is a no-op if we pass the same version as what's already installed.

By default, yum installs the latest version of containerd that's compatible with our pinned dockerd version.